src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import {useParams } from "react-router-dom";
+import {fetchArticleById} from '../api';
+import moment from 'moment';
+
+
+export default function ArticlePage() {
+  
+  const [article, setArticle] = useState({});
+  const { article_id } = useParams();
```

Comment:
Great to see useParams here so I can manualy change the url and the article still loads up!